python/src/aac/plugins/validators/_validator_findings.py

```diff
@@ -59,3 +68,10 @@ def get_info_findings(self) -> list[ValidatorFinding]:
     def _get_findings_with_severity(self, severity: FindingSeverity) -> list[ValidatorFinding]:
         """Return a list of findings of the specified severity."""
         return [finding for finding in self.findings if finding.severity == severity]
+
+    def _get_last_finding(self) -> ValidatorFinding:
+        return self._convert_finding_to_log_message(self.findings[-1])
+
+    def _convert_finding_to_log_message(self, finding: ValidatorFinding) -> str:
+        loc = finding.location
+        return f"Validation {finding.severity.name} failed for definition '{finding.definition.name}' from '{loc.validation_name}' in {loc.source.uri}:{finding.location.location.line}. {finding.message}"
```

Comment:
As mentioned above, I think this should be further from the internals. If I want my finding/log messages to be formatted differently or if I want to include different/additional information it would be challenging, like this.